jerry-core/vm/opcodes.c

```diff
@@ -153,7 +153,16 @@ opfunc_set_accessor (bool is_getter, /**< is getter accessor */
                      ecma_value_t accessor) /**< accessor value */
 {
   ecma_object_t *object_p = ecma_get_object_from_value (object);
-  ecma_string_t *accessor_name_p = ecma_get_string_from_value (accessor_name);
```

Comment:
I think it would be better if we'd call `ecma_op_to_string` every time and add an assert to check the type of accessor_name. Something like this:

```C++
JERRY_ASSERT (ecma_is_value_string (accessor_name) || ecma_is_value_number (accessor_name));
ecma_string_t *accessor_name_p = ecma_get_string_from_value (ecma_op_to_string (accessor_name));
```

In this case there is no need to check the type before the `ecma_deref_ecma_string (accessor_name_p)` call.